website/docs/agent-installation.md

```diff
@@ -11,13 +11,21 @@ In Litmus the Agents can be classified as two types
 
 As part of Litmus installation by default, a self cluster would be registered as Agent in the Portal. From Portal you induce chaos into self cluster and observe the results from the Portal.
 
+<<<<<<< HEAD
```

Comment:
Looks like the conflict is not resolved properly @Jonsy13 do you want to take a look?

Reply:
Sure, thanks. Let me check it.